textile/notes-in-relation-to-move-from-ably-docs.md

```diff
@@ -71,3 +71,152 @@ And the merge commit then created:
+https://github.com/ably/specification/issues/75
+Message: Something went wrong while executing your query. Please include `CB5D:5B59:962B05:98392D:633ACD0C` when reporting this issue., Locations: []
+Message: Something went wrong while executing your query. Please include `CB5E:B2EB:8ECEEC:90E528:633ACD0E` when reporting this issue., Locations: []
+Message: Something went wrong while executing your query. Please include `CB5F:57ED:8FB458:91C1EA:633ACD0F` when reporting this issue., Locations: []
+Message: Something went wrong while executing your query. Please include `CB60:85E8:A5E3AC:A80001:633ACD10` when reporting this issue., Locations: []
+?1 docs %
+```
+
+Clearly some failures, with 11 issues remaining in `ably/docs` after the script completed:
+
+```
+docs % gh issue list --label client-lib-spec --limit 100
+
+Showing 11 of 11 issues in ably/docs that match your search
+
+#435  40140 Timestamp not current ideas                                                        client-lib-spec, knowledge-base, no-sync                  about 2 months ago
+#425  Show a warning when invalid options are used in dynamic languages                        content-request, client-lib-spec, no-sync                 about 2 months ago
+#391  [Proposal] Channel events received on non-attached channels                              content-request, client-lib-spec, no-sync                 about 2 months ago
+#359  Only log outcome from client library retries to fallback hosts                           content-request, client-lib-spec, no-sync                 about 8 months ago
+#332  [Proposal] Add echoMessages flag at the channel level                                    client-lib-spec, no-sync                                  about 2 months ago
+#280  Add requirement to the spec that new fields should not break existing libraries          content-request, client-lib-spec, no-sync                 about 2 months ago
+#263  Add support for strict flag for tokenRequests                                            content-request, api-reference, client-lib-spec, no-sync  about 2 months ago
+#202  Consider mechanism to dynamically change next retry for disconnected / suspended states  content-request, client-lib-spec, no-sync                 about 2 months ago
+#149  Spec: Ensure terminal states have no effect on listeners                                 client-lib-spec, no-sync                                  about 2 months ago
+#141  Add spec for lexicographic ordering                                                      client-lib-spec, no-sync                                  about 2 months ago
+#36   Bundling & queueing                                                                      content-request, client-lib-spec, no-sync                 about 2 months ago
+```
+
+My working theory was that the failures were perhaps due to rate limiting.
+I suspect that may have been the case, as simply running thru the whole process again from the top resulted in success:
```

Comment:
> running thru the whole process again

I hate to be that person, but "through"? ;)

Reply:
As I wrote that sentence I imagined someone would comment.
Serves me right for trying to sound "cool". 😆 

Fixed in https://github.com/ably/specification/pull/87/commits/39ff1461b392eda115e391e67a9209aadb8a5f94.